presto-main/src/main/java/com/facebook/presto/sql/gen/InCodeGenerator.java

```diff
@@ -116,6 +119,8 @@ public BytecodeNode generateExpression(Signature signature, BytecodeGeneratorCon
 
         Signature hashCodeSignature = internalOperator(HASH_CODE, BIGINT, ImmutableList.of(type));
         MethodHandle hashCodeFunction = generatorContext.getRegistry().getScalarFunctionImplementation(hashCodeSignature).getMethodHandle();
+        Signature isIndeterminateSignature = internalOperator(INDETERMINATE, BOOLEAN, ImmutableList.of(type));
```

Comment:
The right way to do this is to use `registry.resolveOperator`. You can change the line above as well.